src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -105,13 +116,26 @@ void* LoadCoreClr(std::string& runtimeDirectory)
         runtimeDirectory = coreClrEnvVar;
 
         std::string coreClrDllPath = runtimeDirectory;
```

Comment:
I don't know - but if writing the error to the console succeeded then it could be helpful.